Lib/ufo2ft/filters/explodeColorLayerGlyphs.py

```diff
@@ -0,0 +1,51 @@
+from ufo2ft.filters import BaseFilter
+from ufo2ft.util import _GlyphSet
+
+
+COLOR_LAYER_MAPPING_KEY = "com.github.googlei18n.ufo2ft.colorLayerMapping"
+COLOR_LAYERS_KEY = "com.github.googlei18n.ufo2ft.colorLayers"
+
+
+class ExplodeColorLayerGlyphsFilter(BaseFilter):
+
+    def start(self):
+        self._layerGlyphSets = {}
+
+    def set_context(self, font, glyphSet):
+        self.globalColorLayerMapping = font.lib.get(COLOR_LAYER_MAPPING_KEY)
+        font.lib[COLOR_LAYERS_KEY] = {}
+        return super().set_context(font, glyphSet)
```

Comment:
the set_context returns a namespace object. you could call the super().set_context at the beginning of your subclass method, get the `ctx` object and set attributes on it, then return it.
The `self.globalColorLayerMapping` attribute should be an attribute of this `self.context` rather than an attribute of `self`, because it changes at every font processed.
does it make sense?

Reply:
Yep, thanks for explaining this, it's much better now.